tests/integration/default/converge.yml

```diff
@@ -0,0 +1,10 @@
+---
+---
```

Comment:
Probably only need one of these

Reply:
Good catch, the error when that happens, well it isn't very helpful.